include/libtorrent/aux_/ip_notifier.hpp

```diff
@@ -34,50 +34,21 @@ POSSIBILITY OF SUCH DAMAGE.
 #define TORRENT_IP_NOTIFIER_HPP_INCLUDED
 
 #include <functional>
-#include <array>
+#include <memory>
 
 #include "libtorrent/error_code.hpp"
-#include "libtorrent/address.hpp"
 #include "libtorrent/io_service.hpp"
 
-#if defined TORRENT_BUILD_SIMULATOR
-// TODO: simulator support
-#elif TORRENT_USE_NETLINK
-#include "libtorrent/netlink.hpp"
-#elif defined TORRENT_WINDOWS
-#include "libtorrent/aux_/disable_warnings_push.hpp"
-#include <boost/asio/windows/object_handle.hpp>
-#include "libtorrent/aux_/disable_warnings_pop.hpp"
-#endif
-
 namespace libtorrent { namespace aux
 {
-	struct ip_change_notifier
+	struct TORRENT_EXTRA_EXPORT ip_change_notifier
```

Comment:
``TORRENT_EXTRA_EXPORT`` would only be necessary for a test to use this class. I can't think of a way to have an automated test for this though, so I'm not sure why this would be here